packages/admin-next/dashboard/src/routes/orders/returns-create/components/create-returns/return-item.tsx

```diff
@@ -0,0 +1,156 @@
+import React from "react"
+import { useTranslation } from "react-i18next"
+
+import { Input, Select, Text } from "@medusajs/ui"
+import { LineItem } from "@medusajs/medusa"
+import { UseFormReturn } from "react-hook-form"
+import { useAdminReturnReasons } from "medusa-react"
+
+import { MoneyAmountCell } from "../../../../../components/table/table-cells/common/money-amount-cell"
+import { Thumbnail } from "../../../../../components/common/thumbnail"
+import { ActionMenu } from "../../../../../components/common/action-menu"
+import { Form } from "../../../../../components/common/form"
+
+type OrderEditItemProps = {
+  item: LineItem
+  currencyCode: string
+
+  form: UseFormReturn<any>
+}
+
+function ReturnItem({ item, currencyCode, form }: OrderEditItemProps) {
+  const { t } = useTranslation()
+
+  const { return_reasons = [] } = useAdminReturnReasons()
+
+  return (
+    <div className="bg-ui-bg-subtle shadow-elevation-card-rest my-2 rounded-xl ">
+      <div className="flex gap-x-2 border-b p-3 text-sm">
+        <div className="flex flex-grow items-center gap-x-3">
+          <Thumbnail src={item.thumbnail} />
+          <div className="flex flex-col">
+            <div>
+              <Text className="txt-small" as="span" weight="plus">
+                {item.title}
+              </Text>
+              {item.variant.sku && <span>(item.variant.sku)</span>}
+            </div>
+            <Text as="div" className="text-ui-fg-subtle txt-small">
+              {item.variant.title}
+            </Text>
+          </div>
+        </div>
+
+        <div className="text-ui-fg-subtle txt-small flex flex-shrink-0">
+          <MoneyAmountCell currencyCode={currencyCode} amount={item.total} />
+        </div>
+
+        <div className="flex items-center">
+          {/*<ActionMenu*/}
+          {/*  groups={[*/}
+          {/*    {*/}
+          {/*      actions: [*/}
+          {/*        {*/}
+          {/*          label: t("actions.remove"),*/}
+          {/*          icon: <Trash />,*/}
+          {/*          // onClick: () => onRemove(item.id),*/}
+          {/*        },*/}
+          {/*      ],*/}
+          {/*    },*/}
+          {/*  ]}*/}
+          {/*/>*/}
+        </div>
+      </div>
+
+      <div className="block p-3 text-sm">
+        <div className="flex justify-between gap-x-2">
```

Comment:
missing responsive styles and spacing is off by 4px. I would suggest changing it to a grid:

```
className="grid grid-cols-1 md:grid-cols-3 gap-3"
```